packages/modul-components/src/filters/phone/phone.ts

```diff
@@ -0,0 +1,32 @@
+import { parsePhoneNumberFromString, PhoneNumber } from 'libphonenumber-js';
+import { PluginObject } from 'vue';
+import { PHONE_NAME } from '../filter-names';
+
+/**
+ * Available formats:
+ * NATIONAL — Example: "(213) 373-4253"
```

Comment:
On pourrait ajouter le lien vers le issue pour éviter à d'autres de repasser à travers tout notre questionnement :) (https://issuetracker.google.com/issues/153616208).

Reply:
J'ai rajouté ça:

* A ticket has been opened by all-time greatest analyst for Google in order to fix the National format. It shouldn't have parenthesis.
 * https://issuetracker.google.com/issues/153616208)